### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Catalin12**

### CODE REVIEW

File: tasks/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+[*]
+charset = utf-8
+indent_style = tab
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.ts]
+quote_type = single
```
Comment (danielursu) [inline tasks/.editorconfig]:
```suggestion
quote_type = double
```

File: tasks/src/app/app.component.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'tasks';
```
Comment (danielursu) [inline tasks/src/app/app.component.ts]: don't need anymore
```suggestion
```

File: tasks/src/app/simple-button/simple-button.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'simple-button',
+    templateUrl: './simple-button.component.html',
+    styleUrls: ['./simple-button.component.css']
+})
+export class SimpleButtonComponent implements OnInit {
+
+    constructor() { }
```
Comment (danielursu) [inline tasks/src/app/simple-button/simple-button.component.ts]:
```suggestion
    public constructor() { }
```

File: tasks/src/app/simple-button/simple-button.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+    ngOnInit(): void {
```
Comment (danielursu) [inline tasks/src/app/simple-button/simple-button.component.ts]:
```suggestion
    public ngOnInit(): void {
```

File: tasks/src/app/simple-button/simple-button.component.ts (hunk 3 above)
Comment (danielursu) [reply tasks/src/app/simple-button/simple-button.component.ts]: let's add access modifiers

File: tasks/src/index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```
Comment (danielursu) [inline tasks/src/index.html:1]: please format

**Review by danielursu — CHANGES_REQUESTED** [review]

File: tasks/src/app/app.component.ts (hunk 2 above)
Comment (Catalin12) [reply tasks/src/app/app.component.ts]: it is necessary to have a title, I think ❔

Comment (danielursu) [reply tasks/src/app/app.component.ts]: in this case you don't use this variable anywhere

Comment (Catalin12) [reply tasks/src/app/app.component.ts]: it s used in app.component.spec.ts
Delete the sequence from there?

Comment (danielursu) [reply tasks/src/app/app.component.ts]: delete the entire spec file

Comment (Catalin12) [reply tasks/src/app/app.component.ts]: ok, all good 😄

**Review by danielursu — APPROVED** [review]

**Pull request merged**